docs/install.rst

```diff
@@ -28,6 +28,19 @@ This will install the latest stable release of Charm4Py, using the default under
 use a specific Charm++ build, you can install and build Charm4Py from source. Note that the source distribution
 is available via "pip install", but the standard from source build process is via "git clone", as outlined below.
 
+Charm4Py provides a small number of examples with the binary distribution that can be used as a sanity check to verify basic functionality of the installation.
+Examples can be run via the following command line tool installed with Charm4Py::
+
+    $ charm4py_test [example]
```

Comment:
Does this `charm4py_test` tool already exist?

Reply:
This PR creates the `charm4py_test` script. I couldn't find any pre-existing script with the same name, if thats what you're wondering?